src/ec/clevo/it5570/acpi/cmos.asl

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+OperationRegion (CMS1, SystemIO, 0x72, 0x2)
+Field (CMS1, ByteAcc, NoLock, Preserve)
+{
+	IND1, 8,
+	DAT1, 8,
+}
+
+IndexField (IND1, DAT1, ByteAcc, NoLock, Preserve)
+{
+	Offset (0x80),
```

Comment:
This is baaad. The first CMOS bank 0x70/0x71 has 128 bytes (0x00-0x7f). Any CMOS access to index above 128 is actually subtracted by 128 and CMOS I/O port pair used should be 0x72/0x73. Soooo... If these settings are starting at bit 1024 (the 128th byte) that means this is offset 0 of the 2nd CMOS bank (0x72/0x73).

Reply:
Ack, copy-paste error from https://github.com/Dasharo/coreboot/blob/95af254e116802490481c6293ae752f5b61e561b/src/ec/starlabs/merlin/acpi/cmos.asl#L19-L33

Reply:
Fixed